packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -6,6 +6,7 @@ import 'dart:math' as math;
 
 import 'package:flutter/painting.dart';
 import 'package:flutter/rendering.dart';
+import 'package:flutter/src/material/floating_action_button_theme.dart';
```

Comment:
I wonder if we should have an umbrella header for themes. Or if material should be exporting them.

Reply:
Would it be a breaking change if we tried to separate them?